stable/jenkins/values.yaml

```diff
@@ -41,9 +41,11 @@ Master:
 #   -Dcom.sun.management.jmxremote.authenticate=false
 #   -Dcom.sun.management.jmxremote.ssl=false
 # JMXPort: 4000
+# Optionally configure a BuildInfoProxy port for artifactory plugin
+# BuildInfoProxyPort: 9000
 # List of plugins to be install during Jenkins master start
   InstallPlugins:
-      - kubernetes:0.11
+      - kubernetes:0.12
```

Comment:
Can we move the update of the plugin to a separate PR? 

Looks like this also causes changes to the config.xml that makes it hard to decipher what is needed. Additionally it was not mentioned in the description so not sure if this is required/intended for the changes listed there.